### DISCUSSION TITLE
usgs-datasheets/

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#30] [Announcements]
# usgs-datasheets/

http://localhost:4000/usgs-datasheets/

**Comment by jengalas** [comment]
Posted by Jeff Olsen
March 22, 2022 at 5:31 am
_Imported from WordPress_

Zhanna, have you tried to get USGS to scan & put online its own datasheets? I have tried to get Kari Craun, former Director of the U.S. Geological Survey, National Geospatial Technical Operations Center (NGTOC) to do so but to no avail. Similar with Juliana Blackwell, NGS director, for NGS to scan its collection of the USGS datasheets.
I can offer the datasheets scanned by the Rolla USGS office for the eastern states, some of which you already have.
Would you be inclined to write to Deb Haaland, Secretary of Interior, requesting this service? I just did so, have not heard back.
I retired from NGS in 2013 after 23 years. Still looking for control marks.
